content/en/docs/best-practices/skywalking-solution-for-openfunction.md

```diff
@@ -0,0 +1,133 @@
+| tags               | A collection of key-value pairs for Span custom tags in tracing. |                        |
+| tags.func          | The name of function. It will be automatically filled.       | "function-a"           |
+| tags.layer         | Indicates the type of service being tracked. It should be set to "faas" when you use the function. | "faas"                 |
+| baggage            | A collection of key-value pairs, exists in the tracing and also needs to be transferred across process boundaries. |                        |
+
+The following is a JSON formatted configuration reference that guides the formatting structure of the tracing configuration.
+
+```json
+{
+  "enabled": true,
+  "provider": {
+    "name": "skywalking",
+    "oapServer": "skywalking-oap:11800"
+  },
+  "tags": {
+    "func": "function-a",
+    "layer": "faas",
+    "tag1": "value1",
+    "tag2": "value2"
+  },
+  "baggage": {
+    "key": "key1",
+    "value": "value1"
+  }
+}
+```
+
+## Enable Tracing Configuration of OpenFunction
+
+### Option 1: global configuration
+
+This document uses `skywalking-opa.default:11800` as an example of the skywalking-opa address in the cluster.
+
+1. Run the following command to modify the configmap `openfunction-config` in the `openfunction` namespace.
+
+   ```shell
+   kubectl edit configmap openfunction-config -n openfunction
+   ```
+
+2. Modify the content under `data.plugins.tracing` by referring to the following example and save the change.
+
+   ```yaml
+   data:
+     plugins.tracing: |
+       enabled: true
+       provider:
+         name: "skywalking"
+         oapServer: "skywalking-oap:11800"
+       tags:
+         func: tracing-function
+         layer: faas
+         tag1: value1
+         tag2: value2
+       baggage:
+         key: "key1"
+         value: "value1"
+   ```
+
+### Option 2: function-level configuration
+
+To enable tracing configuration in the function-level, add the field `plugins.tracing` under `metadata.annotations` in the function manifest as the following example.
+
+```yaml
+metadata:
+  name: tracing-function
+  annotations:
+    plugins.tracing: |
+      enabled: true
+      provider:
+        name: "skywalking"
+        oapServer: "skywalking-oap:11800"
+      tags:
+        func: tracing-function
+        layer: faas
+        tag1: value1
+        tag2: value2
+      baggage:
+        key: "key1"
+        value: "value1"
+```
+
+It is recommended that you use the global tracing configuration, or you have to add function-level tracing configuration for every function you create.
+
+## Use SkyWalking as a Distributed Tracing Solution
+
+1. Create functions by referring to [this document](../interact-with-dapr-output-binding/).
+
+2. Then, you can observe the flow of entire link on the SkyWalking UI.
+
+   ![](/images/docs/en/best-practices/skywalking-solution-for-openfunction/tracing-topology.gif)
```

Comment:
I think it would be better to use the SkyWalking v9 UI in the latest screenshots. It's not a big deal, it could be iterated later.

Reply:
OK.